src/odm_sharing/private/trees.py

```diff
@@ -45,6 +45,13 @@ class NodeKind(StrEnum):
     FILTER = 'filter'
     FIELD = 'field'
     LITERAL = 'literal'
+    RANGE_KIND = 'range-kind'
```

Comment:
Can you add information about this node kind to the technical spec?

Reply:
done